manifests/pipelines/s3-fetch-pipeline.yaml

```
@@ -318,15 +350,21 @@ spec:
       workspace: build-workspace-pv
   - name: skopeo-copy
     params:
-    - name: srcTLSverify
-      value: "true"
-    - name: destTLSverify
-      value: "true"
+    - name: SOURCE_IMAGE_URL
+      value: ""
+    - name: DESTINATION_IMAGE_URL
+      value: ""
     runAfter:
     - retrieve-build-image-info
     taskRef:
-      kind: ClusterTask
-      name: skopeo-copy
+      resolver: git
+      params:
+        - name: url
+          value: https://github.com/openshift-pipelines/tektoncd-catalog
+        - name: revision
+          value: p
+        - name: pathInRepo
+          value: tasks/task-skopeo-copy/0.4.2/task-skopeo-copy.yaml
     workspaces:
     - name: images-url
```

```suggestion
    - name: images_url
```
As per [documentation](https://github.com/openshift-pipelines/tektoncd-catalog/tree/p/tasks/task-skopeo-copy/0.4.2#workspace).
This now needs to be reverted back as we use the new task and not the ClusterTask.

---

Ah okay, makes sense! I was looking into logs but couldn't find anything. I'll update these changes.

---

@biswassri You can see the log [here](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/opendatahub-io_ai-edge/290/pull-ci-opendatahub-io-ai-edge-main-test-ai-edge/1841494657213665280/artifacts/test-ai-edge/test-ai-edge/artifacts/s3-fetch-bike-rentals-auto-ml-s78w6/). 
Search for `failed` in the taskRuns.txt.